app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -55,4 +47,18 @@ def context_authorize!
     authorize! :dashboard,
       context: {organization: Current.organization}
   end
+
+  def calculate_overdue_tasks
```

Comment:
I think we can move this to a controller concern to keep this controller thin. We can call it something like `PetTaskable` or whatever you thin works. I am thinking it will be a concern to make queries on Tasks in the scope of a pet. Then we can add this method there

Reply:
Moved it into the `PetTaskable.rb` concern